src/app/game/components/Player.tsx

```
@@ -1,46 +1,90 @@
+                  // disabled={
+                  //   //TODO : this is not totally correct, later should change it
+                  //   (currentPlayer.marketingCards.length !== 0 &&
+                  //     currentPlayer.turnStatus === "marketing") ||
+                  //   currentPlayer.turnStatus === "planting"
+                  // }
+                >
+                  Add Cards To Hand
+                </Button>
+              </div>
+            )}
+          </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)